test/enabled/query_jobs/kb_Metrics_server_query_jobs_test.py

```diff
@@ -12,6 +12,10 @@
 
 
 class kb_Metrics_query_jobs_Test(Test):
+    def getContext(self):
+        ctx = self.__class__.ctx
+        ctx.update({'user_id': 'eapearson'})
```

Comment:
Because only 'eapearson' could pass the unit tests, this username appears in database files.